gapit benchmark panic

`gapit benchmark --dumptrace foo.gfxtrace --api vulkan de.saschawillems.vulkanBloom` leads to:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa90f90]

goroutine 1 [running]:
github.com/google/gapid/core/app/crash.Crash(0xc54500, 0x13aadf0)
	core/app/crash/crash.go:89 +0xa7
github.com/google/gapid/core/app.Run.func1()
	core/app/run.go:161 +0x62
panic(0xc54500, 0x13aadf0)
	GOROOT/src/runtime/panic.go:513 +0x1b9
main.(*benchmarkVerb).Run.func2(0xc0002ca240, 0xc0002780a0, 0xc000159b18, 0xc000234a00, 0xc000306070, 0xc000306000, 0xc00010dee0)
	cmd/gapit/benchmark.go:128 +0x70
main.(*benchmarkVerb).Run(0xc000234a00, 0xe14f40, 0xc00024f0b0, 0xc000278200, 0x0, 0x0, 0x1, 0xc00024f1a0, 0xc00024e870, 0xc0001684d0, ...)
	cmd/gapit/benchmark.go:159 +0x1ac6
github.com/google/gapid/core/app.(*Verb).Invoke(0x1406360, 0xe14f40, 0xc00024f0b0, 0xc000168480, 0x6, 0x8, 0xc0002782c0, 0xc0002887e0)
	core/app/verbs.go:92 +0x36a
github.com/google/gapid/core/app.VerbMain(0xe14f40, 0xc00024f0b0, 0xc000277d80, 0xe14f40)
	core/app/verbs.go:123 +0x7b
github.com/google/gapid/core/app.Run(0xd7ce10)
	core/app/run.go:253 +0x3ea
main.main()
	cmd/gapit/main.go:24 +0x75
```

--- a/cmd/gapit/benchmark.go
+++ b/cmd/gapit/benchmark.go
@@ -125,8 +125,11 @@ func (verb *benchmarkVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		defer func() {
 			stopGapitTrace()
 			stopGapisTrace()
-			if err := writeTrace(verb.DumpTrace, gapisTrace, gapitTrace); err != nil {
-				log.E(ctx, "Failed to write trace: %v", err)
+			// writeTrace may not be initialized yet
+			if writeTrace != nil {
+				if err := writeTrace(verb.DumpTrace, gapisTrace, gapitTrace); err != nil {
+					log.E(ctx, "Failed to write trace: %v", err)
+				}
 			}
 		}()
 	}

